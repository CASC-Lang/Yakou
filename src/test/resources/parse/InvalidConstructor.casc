class InvalidConstructor

impl InvalidConstructor {
    new() : pub()

    new(a: i32)

    new(b: i32)
}