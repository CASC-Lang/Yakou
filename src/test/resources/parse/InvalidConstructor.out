error: Unexpected token, expected `super` or `self` keyword
--> ./InvalidConstructor.casc:4:12
3 | impl InvalidConstructor {
4 |     new() : pub()
                ^^^
5 | }